sdk/iot/Azure.Iot.Hub.Service/src/ConfigurationsClient.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Iot.Hub.Service.Models;
+
+namespace Azure.Iot.Hub.Service
+{
+    /// <summary>
+    /// Configuration client for automatic device/module management (ADM)
+    /// See <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-automatic-device-management"> Getting started with automatic device management</see>.
+    /// </summary>
+    public class ConfigurationsClient
+    {
+        private readonly ConfigurationRestClient _configurationRestClient;
+
+        /// <summary>
+        /// Initializes a new instance of ConfigurationsClient.
+        /// </summary>
+        protected ConfigurationsClient()
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of ConfigurationsClient.
+        /// <param name="configurationRestClient"> The REST client to perform configurations operations. </param>
+        /// </summary>
+        internal ConfigurationsClient(ConfigurationRestClient configurationRestClient)
+        {
+            Argument.AssertNotNull(configurationRestClient, nameof(configurationRestClient));
+
+            _configurationRestClient = configurationRestClient;
+        }
+
+        /// <summary>
+        /// Gets a configuration on the IoT Hub for automatic device/module management
```

Comment:
Gets a twin configuration ?

Reply:
+1 to this point. If the return type is TwinConfiguration, then the comment should say "twin configuration" not just "configuration"

Reply:
Sure will change. Does the same apply to other APIs?